Networking/NetworkingTests/Remote/ProductsRemoteTests.swift

```diff
@@ -914,11 +914,44 @@ final class ProductsRemoteTests: XCTestCase {
 
         // When/Then
         await assertThrowsError({
-            let _ = try await remote.loadAllSimpleProductsForPointOfSale(for: sampleSiteID)
+            let _ = try await remote.loadSimpleProductsForPointOfSale(for: sampleSiteID)
         }, errorAssert: {
             $0 as? NetworkError == .notFound()
         })
     }
+
+    func test_loadAllSimpleProductsForPointOfSale_when_page_has_products_then_loads_expected_products() async throws {
```

Comment:
super nit: using the new function name (same for the other test case)
```suggestion
    func test_ loadSimpleProductsForPointOfSale_when_page_has_products_then_loads_expected_products() async throws {
```

Reply:
Definitely! https://github.com/woocommerce/woocommerce-ios/pull/14187/commits/8ea205360704409ec7b9d75e30c2c37606bf028c